scripts/deploy-docs.sh

```diff
@@ -24,10 +29,16 @@ cd ../
 git clone $THEME_REPO
 
 mkdocs build --clean
-
-cd out
-
-touch .
-git add -A .
-git commit -m "GH-Pages update by travis after $TRAVIS_COMMIT"
-git push -q origin master 
+build_result=$?
+
+# Only deploy after merging to master
+if [ "$TRAVIS_PULL_REQUEST" == "false" -a "$TRAVIS_BRANCH" == "master" ]; then
```

Comment:
Never thought about doing it this way, nice work :)

Would it be a good idea to check the build_result first and if it's not 0 then exit before checking travis variables?

Reply:
probably, no point in pushing broken docs.